docs/gettsim_objects/input_variables.md

```diff
@@ -546,3 +546,15 @@ Type: int
 Number of own children below the age of 25 (incl. not in hh)
 
 Type: int
+
+## `budget_erz_geld`
+
+Dummy: applied for "Budgetsatz" of parental leave benefit
+
+Type: bool
+
+## `inanspruchn_erzgeld`
```

Comment:
Maybe we need a general discussion about these claim variables. If those are true and then gettsim calculates that individuals are not eligible. How do we handle that?

Reply:
The interpretation can only ever be conditional on GETTSIM deciding that someone is eligible. Anything else would be measurement error or GETTSIM needs refinement.

Reply:
However, thinking of it, I think we should get rid of it for now. People get the transfer if they qualify, period. Else you'll need to remove erziehungsgeld from the targets. More general handling of take up is in the making.

Btw, please write out: `erziehungsgeld`.  The two abbreviations here are inconsistent with each other already.